### DISCUSSION TITLE
Not Moving to the Next Steps ?? From Step 2

### DISCUSSION THREAD

**Discussion by jenushanjesuthasan** [skills/.github#539] [Communicate using Markdown]
https://github.com/jenushanjesuthasan/skills-communicate-using-markdown/blob/start-markdown/index.md

I added the image and preview and commit the changes, but still it doesn't move to the next step

**Comment by Petratype** [comment]
im stuck in the first! i don't know why it happens, i did all the activities and still not refreshes

**Reply by mjbear** [reply]
> im stuck in the first! i don't know why it happens, i did all the activities and still not refreshes

@Petratype
What branch are you working on within your fork of this repo?

My suggestion above [might help](https://github.com/orgs/skills/discussions/539#discussioncomment-10313845)?
